services/send_user_report.py

```diff
@@ -15,7 +15,7 @@ def get(self):
         self.response.headers['Content-Type'] = 'application/json'
         rate = self.request.get('rate')
         report_rate = int(rate)
-        email_address = "info@connexusapt.com"
+        email_address = "rayolanderos@gmail.com"
```

Comment:
Can you use `user.email()` to make the user responsible for inviting people? Or is not accessible here?

Reply:
It isn't available as the cron doesn't run this as an auth user. I added myself as an Authorized sender in my deployment to test. It works, but just ran into the issue of Email Quotas. We get 10 a day and I already max-ed out mine.  -- Following up in Slack